src/pages/index.js

```diff
@@ -16,24 +16,36 @@ const formValidatorConfig = {
 };
 
 /*Elements*/
-
+const avatarEditButton = document.querySelector("#avatar-edit-button");
 const profileEditButton = document.querySelector("#profile-edit-button");
 const profileEditModal = document.querySelector("#profile-edit-modal");
 const addCardModal = document.querySelector("#add-form-modal");
+const avatarEditModal = document.querySelector("#avatar-edit-modal");
 const addNewCardButton = document.querySelector("#profile-add-button");
 const profileTitleInput = document.querySelector("#profile-title-input");
 const profileDescriptionInput = document.querySelector(
   "#profile-description-input"
 );
+const avatarEditForm = avatarEditModal.querySelector(".modal__form");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms